common/src/main/java/edu/snu/nemo/common/test/ArgBuilder.java

```diff
@@ -19,12 +19,14 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.StringJoiner;
-import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 /**
  * Argument builder.
  */
 public final class ArgBuilder {
+  private static final List<List<String>> DEFAULT_ARGS = Arrays.asList(Arrays.asList("-executor_json",
```

Comment:
Is this needed to override the json path?
If so, can't we bind parameter through Tang instead of this?

Reply:
I'd rather not fix it, since it would prevent users from running their applications using the `bin/run.sh` script without explicitly setting the `executor_json` parameter. This is indeed the default value for running integration tests, but not for all use cases.

Reply:
I agree with @sanha on this.

Reply:
It's quite difficult, since Tang appears after this step, and it created duplication of code, which will later require refactoring in some other way. I've cleaned up the code a bit to make it look more neat.